packages/interflex/README.md

```diff
@@ -0,0 +1,3 @@
+# interflex
```

Comment:
Personally, I think we shouldn't abuse "flex" 😆 

Oh, while on the topic of name, since this is a new scope/package in this repo, let's add the name to `.github/semantic.yml` so the status check for the title could pass. Poke @danielsukmana 🍡

Reply:
`transl8` is lit 🔥 

Thanks for the pointer @darcien !

Reply:
Yeah, `transl8` pretty lit